packages/cra-template/template.json

```diff
@@ -0,0 +1,19 @@
+{
+  "package": {
+    "dependencies": {
+      "@fluentui/react": "^7.155.3",
```

Comment:
Not a blocker but it would be nice if there was a way to automatically update these as new versions are published.

Reply:
what do you mean? like when we release new react version it should be bumped here as well? 

anyways I think that's out of scope for this PR

Reply:
Yeah that was what I meant. I'm fine with leaving it as-is and then revisiting if it causes problems.